source/mir/ndslice/slice.d

```diff
@@ -1018,25 +1021,21 @@ public:
     {
```

Comment:
The condition should check the labels as well. Check allSatisfy from meta

Reply:
Addressed in 7183830